alert_group_nl_log2slack.py

```diff
@@ -181,18 +185,21 @@ def dump_cookies(session, where):
 def login_and_fetch(klant_nummer, klant_gecrypt):
     # Session keeps cookies around.
     with requests.Session() as session:
-        ret = session.get(ALERTMOBILE_URL)
+        ret = session.get(ALERTMOBILE_URL,
+                          timeout=10)
```

Comment:
This is not the 4 space indent.

And there is no need for this to be on 2 lines.